### PR TITLE
[Locked Figure Aria] Use spoken math in locked figure settings autogen labels

### PR DESCRIPTION
**Pull request by nishasy**
## Summary:
When auto-generating the aria labels for locked figures, we want it to use
words as if they were spoken rather than math expressions that might be
read incorrectly by the screen reader.

- Use the `generateSpokenMathDetails` utility within LockedVectorSettings,
  LockedEllipseSettings, LockedPolygonSettings, and LockedFunctionSettings.
- Updated LockedFigureAria - remove the prop `prePopulatedAriaLabel` and
  make `getPrepopulatedAriaLabel` required.

Issue: https://khanacademy.atlassian.net/browse/LEMS-2548

## Test plan:
- `yarn jest packages/perseus-editor/src/widgets/interactive-graph-editor/locked-figures/locked-vector-settings.test.tsx`
- `yarn jest packages/perseus-editor/src/widgets/interactive-graph-editor/locked-figures/locked-ellipse-settings.test.tsx`
- `yarn jest packages/perseus-editor/src/widgets/interactive-graph-editor/locked-figures/locked-polygon-settings.test.tsx`
- `yarn jest packages/perseus-editor/src/widgets/interactive-graph-editor/locked-figures/locked-function-settings.test.tsx`

Storybook
- Go to http://localhost:6006/iframe.html?args=&id=perseuseditor-widgets-interactive-graph--mafs-with-locked-figure-labels-all-flags&viewMode=story
- For each locked figure:
  - Open the figure settings
  - Change the visible label to have a mix of TeX and non-TeX
  - Press the "Auto-generate" button
  - Confirm that the input changes to include spoken math words for the TeX
  - Also try this with no labels, multiple labels, fully non-TeX, and
    fully teX labels.

### CODE REVIEW

**Review by anakaren-rojas — APPROVED** [review]

**Comment by catandthemachines** [comment]
Was playing around with the experience and I was noticing how it's reading out square roots. Is this expected behavior for it to read out "StartRoot one half EndRoot"? That doesn't feel natural, but maybe I'm not aware of the standard for reading out TeX.
image

File: packages/perseus-editor/src/widgets/interactive-graph-editor/locked-figures/locked-function-settings.test.tsx [hunk 1]
```diff
@@ -24,6 +24,14 @@ const defaultProps = {
 
 const defaultLabel = getDefaultFigureForType("label");
 
+// Mock the async function generateSpokenMathDetails
+jest.mock("./util", () => ({
```
Comment (catandthemachines) [inline packages/perseus-editor/src/widgets/interactive-graph-editor/locked-figures/locked-function-settings.test.tsx:28]: Love this mocking to help us identify the generated spoken math details in testing ❤️

**Review by catandthemachines — APPROVED** [review]
Let's ship it!

**Comment by nishasy** [comment]
The parent pull-request ([#1854](https://github.com/Khan/perseus/pull/1854)) has been merged into `main`, but this branch (`spoken-math-others`) now has conflicts with the new base branch. These conflicts must be resolved before checks can complete on this pull-request.

**Pull request merged**